addons/website_slides/static/src/js/slides_course_fullscreen_player.js

```diff
@@ -370,7 +367,7 @@ odoo.define('website_slides.fullscreen', function (require) {
         */
         _fetchSlideContent: function (){
             var slide = this.get('slide');
-            if (slide.type === 'webpage') {
+            if (slide.type === 'webpage' && !slide.isQuiz) {
```

Comment:
Should probably not be in this commit but I won't tell anyone.

(Never-mind I just told the whole open space)

Reply:
done